examples/nest/hpc_benchmark_invariant.sli

```diff
@@ -0,0 +1,512 @@
+
+  M_INFO (BuildNetwork)
+  (Creating excitatory stimulus generator.) message
+
+  model_params using
+
+  % Convert synapse weight from mV to pA
+  C_m tau_syn tau_m ConvertSynapseWeight /conversion_factor Set
+  /JE_pA conversion_factor JE mul def
+
+  /nu_thresh V_th CE tau_m C_m div mul JE_pA mul 1.0 exp mul tau_syn mul div def
+  /nu_ext nu_thresh eta mul def
+  endusing
+
+  /E_stimulus /poisson_generator Create def
+  E_stimulus
+  <<
+     /rate nu_ext CE mul 1000. mul
+  >> SetStatus
+
+  M_INFO (BuildNetwork)
+  (Creating excitatory spike detector.) message
+
+  record_spikes
+  {
+    /detector_label  filestem (/alpha_) join stdp_params /alpha get cvs join (_spikes) join def
+    /E_detector /spike_detector Create def
+    E_detector
+    <<
+       /withtime true
+       /to_file true
+       /label detector_label
+    >> SetStatus
+  } if
+
+  memory_thisjob cvs ( # virt_mem_after_nodes) join logger /log call
+
+  toc /BuildNodeTime Set
+
+  BuildNodeTime cvs ( # build_time_nodes) join logger /log call
+
+  tic
+
+  % Create custom synapse types with appropriate values for
+  % our excitatory and inhibitory connections
+  /static_synapse_hpc << /delay delay >> SetDefaults
+  /static_synapse_hpc /syn_std  CopyModel
+  /static_synapse_hpc /syn_ex << /weight JE_pA >> CopyModel
+  /static_synapse_hpc /syn_in << /weight JE_pA g mul >> CopyModel
+
+  stdp_params /weight JE_pA put 
+  /stdp_pl_synapse_hom_hpc stdp_params SetDefaults
+
+
+  M_INFO (BuildNetwork)
+  (Connecting stimulus generators.) message
+
+  % Connect Poisson generator to neuron
+
+  E_stimulus E_stimulus cvgidcollection E_neurons << /rule (all_to_all) >> << /model /syn_ex >> Connect
+  E_stimulus E_stimulus cvgidcollection I_neurons << /rule (all_to_all) >> << /model /syn_ex >> Connect
+
+  M_INFO (BuildNetwork)
+  (Connecting excitatory -> excitatory population.) message
+
+  E_neurons E_neurons << /rule (fixed_indegree) /indegree CE /autapses false /multapses true >>
+                      << /model /stdp_pl_synapse_hom_hpc >> Connect
+
+  M_INFO (BuildNetwork)
+  (Connecting inhibitory -> excitatory population.) message
+
+  I_neurons E_neurons << /rule (fixed_indegree) /indegree CI /autapses false /multapses true >>
+                      << /model /syn_in >> Connect
+
+  M_INFO (BuildNetwork)
+  (Connecting excitatory -> inhibitory population.) message
+
+  E_neurons I_neurons << /rule (fixed_indegree) /indegree CE /autapses false /multapses true >>
+                      << /model /syn_ex >> Connect
+
+  M_INFO (BuildNetwork)
+  (Connecting inhibitory -> inhibitory population.) message
+
+  I_neurons I_neurons << /rule (fixed_indegree) /indegree CI /autapses false /multapses true >>
+                      << /model /syn_in >> Connect
+
+
+  record_spikes true eq
+  {
+    E_neurons size Nrec lt
+    {
+      M_ERROR (BuildNetwork)
+      (Networks is smaller than number of neurons to record from!) message
```

Comment:
since your only checking the size of the E population, the error message should reflect this